rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/TclLauncherAction.xtend

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2016 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.rcp4.views.tcltestrun
+
+import javax.inject.Inject
+import org.eclipse.core.resources.IResource
+import org.eclipse.core.runtime.FileLocator
+import org.eclipse.core.runtime.Path
+import org.eclipse.core.runtime.Platform
+import org.eclipse.jface.action.Action
+import org.eclipse.jface.resource.ImageDescriptor
+import org.eclipse.jface.viewers.StructuredSelection
+import org.eclipse.ui.ISelectionService
+import org.testeditor.dsl.common.ui.workbench.PartHelper
+
+class TclLauncherAction extends Action {
+
+	@Inject TclLauncherUi launcherUI
+	@Inject PartHelper partHelper
+
+	protected new() {
+		val bundle = Platform.getBundle("org.testeditor.rcp4.views.tcltestrun")
```

Comment:
wouldn't that be something for the PlatformHelper? Otherwise PlatformHelper looks a bit empty ...